nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -204,6 +214,7 @@ public class ValidateCsv extends AbstractProcessor {
             .description("FlowFiles that are not valid according to the specified schema are routed to this relationship")
             .build();
 
+
```

Comment:
This extra blank line doesn't appear to be necessary.

Reply:
Removed.